apps/www/src/styles/typeset.css

```
@@ -0,0 +1,75 @@
+  --docs-label-tracking: 0.12em;
+  --docs-label-weight: var(--rs-font-weight-medium);
+
+  /* Caption — table footnotes, muted asides. */
+  --docs-caption-size: 13px;
+  --docs-caption-leading: 19px;
+  --docs-caption-tracking: 0em;
+
+  --rs-line-height-mono-small: 20px;
+
+  --rs-font-mono:
+    var(--docs-font-mono), ui-monospace, SFMono-Regular, Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)